scripts/check_changes_in_right_version.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# get the current development version
+CURRENT_VERSION=$(grep 'latestVersion' src/config/ci.ts | cut -d ':' -f 2 | sed "s/'//g" | sed "s/,//g")
```

Comment:
in case we ever change our config format and this breaks from under us, maybe we can have a small unit in the common folder that appends a string to an old version and checks that this fails correctly?